SECURITY.md

```diff
@@ -0,0 +1,106 @@
+4. If a vulnerability is acknowledged and the timeline for a fix is determined, the maintainers will work on a plan to communicate with the appropriate community, including identifying mitigating steps that affected users can take to protect themselves until the fix is rolled out.
+5. The maintainers will also create a [Security Advisory](https://docs.github.com/en/code-security/repository-security-advisories/publishing-a-repository-security-advisory) using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0). The maintainers make the final call on the calculated CVSS; it is better to move quickly than making the CVSS perfect. Issues may also be reported to [Mitre](https://cve.mitre.org/) using this [scoring calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator). The draft advisory will initially be set to private.
+6. The maintainers will work on fixing the vulnerability and perform internal testing before preparing to roll out the fix.
+7. The maintainers will provide early disclosure of the vulnerability by emailing a security mailing list if such exists. Distributors can initially plan for the vulnerability patch ahead of the fix, and later can test the fix and provide feedback to the maintainer team. See the section **Early Disclosure to This Project's Distributors List** for details about how to join this mailing list.
+8. A public disclosure date is negotiated by the maintainers, the bug submitter, and the distributors list. We prefer to fully disclose the bug as soon as possible once a user mitigation or patch is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for distributor coordination. The timeframe for disclosure is from immediate (especially if it’s already publicly known) to a few weeks. For a critical vulnerability with a straightforward mitigation, we expect the report date for the public disclosure date to be on the order of 14 business days. The maintainer team holds the final say when setting a public disclosure date.
+9. Once the fix is confirmed, the maintainers will patch the vulnerability in the next patch or minor release, and backport a patch release into all earlier supported releases. Upon release of the patched version of this project, we will follow the **Public Disclosure Process**.
+
+
+## Public Disclosure Process
+
+The maintainers publish the public advisory to this project's community via GitHub. In most cases, additional communication via Slack, Twitter, mailing lists, blog, and other channels will assist in educating the project's users and rolling out the patched release to affected users.
+
+The maintainers will also publish any mitigating steps users can take until the fix can be applied to their instances. This project's distributors will handle creating and publishing their own security advisories.
+
+
+## Mailing lists
```

Comment:
I think we should cut this whole section and only put it in the security policy doc for the projects that have one (see related comment above).